httpx/models.py

```diff
@@ -683,11 +683,25 @@ def __init__(
             self.is_closed = True
             self.is_stream_consumed = True
             self._raw_content = content or b""
+            self._elapsed = datetime.timedelta(0)
```

Comment:
Shouldn't this be `request.timer.elapsed`? Otherwise we don't account for the time spent receiving the response headers in the raw-bytes case.

Reply:
Ah that'd be better yup. As it *happens* we'll always use `stream=...` for actual network responses, it's really only the "mock response instance" case that we use `content=...` for, but yes it makes sense to account for the time properly in that case too.